config/analytics_hidden_pii.yml

```diff
@@ -0,0 +1,4 @@
+---
+:shared:
```

Comment:
Do we want to use "symbolized" keys here? We don't seem to in `analytics.yml`. I'm guessing the gem is indifferent to this, but we might want to be internally consistent.